### PR TITLE
add vendor script to the commit

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ make vendor-update
 4. Commit updated glide files and vendor
 
 ```bash
-git add glide.yaml glide.lock vendor
+git add glide.yaml glide.lock scripts/vendor-openshift.sh vendor
```
Comment (cdrage) [inline docs/development.md]: Like I mentioned before, wouldn't this all be easier just to do a `git add .` and add a note here not not include and `.go` changes?

**Comment by surajssd** [comment]
@containscafeine yes we need it, see this example for some reference https://github.com/kedgeproject/kedge/pull/506/files#diff-4e4a5d0cc48e6822f990cad4695eb90eR15

File: docs/development.md (hunk 1 above)
Comment (kadel) [reply docs/development.md]: I think that listing files explicitly like this is much safer.

**Comment by surajssd** [comment]
@containscafeine @cdrage I think both of yours questions have been answered, can I merge this one?

**Comment by kadel** [comment]
TIMEOUT - this is small documentation change so I'm merging this.
We can change it to something else if @containscafeine @cdrage still disagree with this.

**Pull request merged**